### PR TITLE
{chem}[intel-compilers/2021.4.0] libxc v5.1.6

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/libxc/libxc-5.1.6-intel-compilers-2021.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+homepage = 'https://www.tddft.org/programs/libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'intel-compilers', 'version': '2021.4.0'}
+
+source_urls = ['https://www.tddft.org/programs/libxc/down.php?file=%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['02e4615a22dc3ec87a23efbd3d9be5bfad2445337140bad1720699571c45c3f9']
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/l/libxc/libxc-5.1.6-intel-compilers-2021.4.0.eb:14]:
```suggestion
checksums = ['0d0496513114ed84987c32649912f9a65d234696dab13a4b00ce8c420bfc0a1d']
```

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/633b06d20e09cac277ea9658d96da412 for a full test report.

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**